src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -1437,48 +1437,6 @@ public static void DCS_RootNameNamespaceAndKnownTypesThroughConstructorAsXmlDict
     }
 
     [Fact]
-    public static void DCS_ExceptionObject()
```

Comment:
@morganbr can you please mark these tests with [ActiveIssue] https://github.com/dotnet/corefx/issues/20361 instead of removing them? It would be a little easier for me to update the tests to test custom exception with these scenarios.